content/en/integrations/guide/azure-portal.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Datadog in the Azure Portal
+kind: guide
+further_reading:
+  - link: "integration/azure"
+    tag: "Documentation"
+    text: "Azure integration"
+  - link: "https://www.datadoghq.com/blog/azure-datadog-partnership"
+    tag: Blog
+    text: "Microsoft partnership embeds Datadog natively in the Azure portal"
+---
+
+{{< site-region region="us,eu,gov" >}}
+<div class="alert alert-warning">
+  This guide is for Datadog customers using US3 with Azure.
+</div>
+{{< /site-region >}}
+
+This guide is for managing the integration between Azure and Datadog in the Azure portal using the Datadog resource. The Datadog resource in Azure represents the connection between a Datadog organization and an Azure subscription. Create a Datadog resource in Azure before proceeding with this guide.
```

Comment:
Let's add a link to the configuration section of the Azure integration page here where it says "create a datadog resource in Azure"